webpack.config.js

```diff
@@ -3,6 +3,29 @@ const webpack = require('webpack');
 const CleanWebpackPlugin = require('clean-webpack-plugin');
 const fs = require('fs');
 
+// A list of languages we support syntax highlighting for.
+// This whitelist is used to omit the inclusion of languages we don't
+// support in order to keep the bundle smaller.
+const hljsSupportedLanguages = [
```

Comment:
Yeah, the only problem with that is that they're exported from a TS file that is built using this script. Not sure how I'd go about sharing these values. Admittedly I didn't spend too much time trying to solve for this, but I did try for a little bit to get that to work.

Reply:
Hmmm....yeah true. We could have this as a .js file in the syntax package so it can be freely imported by both. Curious what that would do to the TS compiler if it was a .JS file.

Reply:
I tried this, and had a lot of issues getting it to work with our TS validation – can't remember the exact issues I had, but I spent a few hours on this. Ended up leaving a comment in the types file in Syntax as a reminder.

Reply:
Meh, alright.

Reply:
🙁